feat: kafka bindings with support for scalar parameter values (#150)
asyncapi/bindings: kafka/README.md

Field Name | Type | Description
---|:---:|---

> <a name="operationBindingObjectGroupId"></a>`groupId` | [Schema Object][schemaObject] \| string | Id of the consumer group.
> <a name="operationBindingObjectClientId"></a>`clientId` | [Schema Object][schemaObject] \| string | Id of the consumer inside a consumer group.

Reviewer: ```suggestion
<a name="operationBindingObjectGroupId"></a>`groupId` | string \| [Schema Object][schemaObject] | Id of the consumer group or a [Schema Object][schemaObject] that validates it.
<a name="operationBindingObjectClientId"></a>`clientId` | [string \| [Schema Object][schemaObject] | Id of the consumer inside a consumer group or a [Schema Object][schemaObject] that validates it.
```

> <a name="operationBindingObjectClientId"></a>`clientId` | [string \| [Schema Object][schemaObject] | Id of the consumer inside a consumer group or a [Schema Object][schemaObject] that validates it.

Reviewer 2: ```suggestion
<a name="operationBindingObjectClientId"></a>`clientId` | string \| [Schema Object][schemaObject] | Id of the consumer inside a consumer group or a [Schema Object][schemaObject] that validates it.
```

Field Name | Type | Description
---|:---:|---

> <a name="messageBindingObjectKey"></a>`key` | [Schema Object][schemaObject] \| [AVRO Schema Object](https://avro.apache.org/docs/current/spec.html) \| string | The message key. **NOTE**: You can also use the [reference object](https://asyncapi.io/docs/specifications/v2.1.0#referenceObject) way.

Reviewer: ```suggestion
<a name="messageBindingObjectKey"></a>`key` | string \| [Schema Object][schemaObject] \| [AVRO Schema Object](https://avro.apache.org/docs/current/spec.html) | The message key or an object that validates the key. **NOTE**: You can also use a [Reference Object](https://asyncapi.io/docs/specifications/v2.1.0#referenceObject).
```

> <a name="messageBindingObjectKey"></a>`key` | string \| [Schema Object][schemaObject] \| [AVRO Schema Object](https://avro.apache.org/docs/current/spec.html) | The message key or an object that validates the key. **NOTE**: You can also use a [Reference Object](https://asyncapi.io/docs/specifications/v2.1.0#referenceObject).

Reviewer 3: ```suggestion
<a name="messageBindingObjectKey"></a>`key` | string \| [Schema Object][schemaObject] \| [AVRO Schema Object](https://avro.apache.org/docs/current/spec.html) | The message key or an object that validates the key. **NOTE**: You can also use a [Reference Object](https://asyncapi.io/docs/specifications/v2.4.0#referenceObject).
```

worth updating this while we're here?

Reviewer 4: +1 for @dalelane suggestion
